### PR TITLE
Rework Horus BL handling

### LINKED ISSUE #5403: Feature Request:  Turn off screen on Horus X10
The original frOS was able to turn off the screen.  Can we get a feature to completely turn off the screen.

Thanks.

### LINKED ISSUE #5621: Feature Request - Horus X10S Backlight off
Hi,
Please would it be possible to have the screen backlight go completely off on a timer like you can in FROS. Currently can only set it to dim to around 50% in Open TX. Many thanks.

### PR DESCRIPTION
**Pull request by 3djc**
- OFF mode is not available for Horus anymore, as it makes no sense for a radio that requires BL
- if OFF brightness is set to the very mimium, then BL is turned off during off periods

This fixes #5621 and fixes #5403

### CODE REVIEW

**Comment by bsongis** [comment]
Tested on X10

File: radio/src/gui/480x272/radio_setup.cpp [hunk 1]
```diff
@@ -360,7 +360,8 @@ bool menuRadioSetup(event_t event)
 
       case ITEM_SETUP_BACKLIGHT_MODE:
         lcdDrawText(MENUS_MARGIN_LEFT, y, STR_MODE);
-        g_eeGeneral.backlightMode = editChoice(RADIO_SETUP_2ND_COLUMN, y, STR_VBLMODE, g_eeGeneral.backlightMode, e_backlight_mode_off, e_backlight_mode_on, attr, event);
+        if (attr & INVERS) g_eeGeneral.backlightMode = checkIncDec(event, g_eeGeneral.backlightMode, e_backlight_mode_keys, e_backlight_mode_on, (menuVerticalPositions[0] == 0) ? EE_MODEL : EE_GENERAL);
```
Comment (bsongis) [inline radio/src/gui/480x272/radio_setup.cpp]: `if (attr & INVERS) g_eeGeneral.backlightMode = checkIncDec(event, g_eeGeneral.backlightMode, e_backlight_mode_keys, e_backlight_mode_on, EE_GENERAL)`

**Comment by Sixx2099** [comment]
Hello 3djc,
will this change only be available in the next OpenTx version, or is there a way to get this change onto a Horus X10 already?

Greetings
Sixx2099

**Comment by kilrah** [comment]
It is in nightly builds since it was merged (but those shouldn't be used for flying).

**Pull request merged**